packages/@aws-cdk/cloud-assembly-schema/lib/cloud-assembly/context-queries.ts

```diff
@@ -492,6 +492,8 @@ export interface PluginContextQuery {
 
   /**
    * Arbitrary other arguments for the plugin
+   *
+   * @internal
    */
   [key: string]: any;
```

Comment:
This is silently ignored by `jsii@v1`, so impossible to use in non-JS/TS languages. I propose to make this `@internal` or to remove it.